src/ng1/plugins/customscroll/jquery.jscrollpane.js

```diff
@@ -136,7 +136,7 @@ LICENSE-END
 											(parseInt(elem.css('paddingRight'), 10) || 0);
 			}
 
-			function initialise(s)
+			function initialise(s, mutationObserved)
```

Comment:
Rename this variable to something like `forceResize` or something as the value of this variable will force it to recalculate the size and may have other uses than only when a mutation is observed.